index.bs

```diff
@@ -234,35 +223,40 @@ interface Operand {};
 
 ## NeuralNetworkContext ## {#api-neuralnetworkcontext}
 
-The {{NeuralNetworkContext}} defines a set of operations derived from the first-wave models [[Models]] that address identified [[#usecases]].
+The {{NeuralNetworkContext}} interface represents a global state of neural network compute workload and execution processes.
 
 <script type=idl>
-typedef double number;
-
-dictionary NamedOperand {
-  required DOMString name;
-  required Operand operand;
+interface NeuralNetworkContext {
+  ModelBuilder createModelBuilder();
 };
+</script>
 
-interface NeuralNetworkContext {
-  // Create an Operand object that represents a model input.
+## ModelBuilder ## {#api-modelbuilder}
+
+The {{ModelBuilder}} interface defines a set of operations as identified by the [[#usecases]] that can be composed into a computational graph. It also represents the intermediate state of a graph building session.
+
+<script type=idl>
+typedef record<DOMString, Operand> Operands;
```

Comment:
`NamedOperands`? `Opearnds` sounds like a list.

Reply:
`NamedInputs` and `NamedOutputs` for consistency?

Reply:
sounds good.